Packs/AWS-SecurityHub/Integrations/AWS_SecurityHub/README.md

```diff
@@ -119,20 +124,20 @@ Used by Security Hub customers to update information about their investigation i
 | roleSessionName | An identifier for the assumed role session. | Optional | 
 | roleSessionDuration | The duration, in seconds, of the role session. The value can range from 900 seconds (15 minutes) up to the maximum session duration setting for the role. | Optional | 
 | raw_json | Override arguments and send a formatted JSON file. | Optional | 
-| finding_identifiers_id | The identifier of the finding that was specified by the finding provider. | Optional | 
-| finding_identifiers_product_arn | The ARN generated by Security Hub that uniquely identifies a product that generates findings. This can be the ARN for a third-party product that is integrated with Security Hub, or the ARN for a custom integration. | Optional | 
+| finding_identifiers_id | The identifier of the finding that was specified by the finding. Can be retrieved using the 'aws-securityhub-get-findings' command provider. | Required | 
+| finding_identifiers_product_arn | The ARN generated by Security Hub that uniquely identifies a product that generates findings. This can be the ARN for a third-party product that is integrated with Security Hub, or the ARN for a custom integration. Can be retrieved using the 'aws-securityhub-get-findings' command. | Required | 
 | note_text | The updated note text. | Optional | 
 | note_updated_by | The principal that updated the note. | Optional | 
-| severity_label | The severity value of the finding. The allowed values are the following. *   INFORMATIONAL - No issue was found. *   LOW - The issue does not require action on its own. *   MEDIUM - The issue must be addressed but not urgently. *   HIGH - The issue must be addressed as a priority. *   CRITICAL - The issue must be remediated immediately to avoid it escalating. | Optional | 
-| verification_state | Indicates the veracity of a finding. The available values for VerificationState are as follows. *   UNKNOWN - The default disposition of a security finding. *   TRUE_POSITIVE - The security finding is confirmed. *   FALSE_POSITIVE - The security finding was determined to be a false alarm. *   BENIGN_POSITIVE - A special case of TRUE_POSITIVE where the finding doesn't pose any threat, is expected, or both. | Optional | 
-| types | One or more finding types in the format of namespace/category/classifier that classify a finding. Valid namespace values are as follows.  Software and Configuration Checks, TTPs, Effects, Unusual Behaviors, Sensitive Data Identifications | Optional | 
+| severity_label | The severity value of the finding. The allowed values are the following. *   INFORMATIONAL - No issue was found. *   LOW - The issue does not require action on its own. *   MEDIUM - The issue must be addressed but not urgently. *   HIGH - The issue must be addressed as a priority. *   CRITICAL - The issue must be remediated immediately to avoid it escalating. Possible values are: INFORMATIONAL, LOW, MEDIUM, HIGH, CRITICAL. | Optional | 
```

Comment:
```suggestion
| severity_label | The severity value of the finding. The allowed values are the following. *   INFORMATIONAL - No issue was found. *   LOW - The issue does not require action on its own. *   MEDIUM - The issue must be addressed but not urgently. *   HIGH - The issue must be addressed as a priority. *   CRITICAL - The issue must be remediated immediately to avoid it escalating.  | Optional | 
```
Adding that line is redundant.  You've already listed all the allowed values.